src/crimson/os/seastore/seastore.cc

```diff
@@ -1457,8 +1457,44 @@ SeaStore::tm_ret SeaStore::_omap_rmkeyrange(
 {
   LOG_PREFIX(SeaStore::_omap_rmkeyrange);
   DEBUGT("{} first={} last={}", *ctx.transaction, *onode, first, last);
-  assert(0 == "not supported yet");
-  return tm_iertr::now();
+  if (first > last) {
+    ERROR("range error, first: {} > last:{}", first, last);
+    abort();
+  }
+  auto omap_root = onode->get_layout().omap_root.get(
+    onode->get_metadata_hint(device->get_block_size()));
+  if (omap_root.is_null()) {
+    return seastar::now();
+  } else {
+    return seastar::do_with(
+      BtreeOMapManager(*transaction_manager),
+      onode->get_layout().omap_root.get(
+        onode->get_metadata_hint(device->get_block_size())),
+      std::move(first),
+      std::move(last),
+      [&ctx, &onode](
+	auto &omap_manager,
+	auto &omap_root,
+	auto &first,
+	auto &last) {
+      auto config = OMapManager::omap_list_config_t();
+      config.inclusive = true;
```

Comment:
@athanatos @rzarzynski do you know for the `OP_OMAP_RMKEYRANGE`, is it supposed to remove the `last` key or not?

Reply:
Ah, I'm not sure, I'd have to check the BlueStore code.

Reply:
I am checking how BlueStore do.

Reply:
it's left closed right open
see https://github.com/ceph/ceph/blob/main/src/os/Transaction.h#L1168